src/test/java/se/sundsvall/supportmanagement/api/MetadataCategoriesResourceTest.java

```diff
@@ -32,22 +32,25 @@ class MetadataCategoriesResourceTest {
 	private int port;
 
 	@Test
-	void createCatgory() {
+	void createCategory() {
 		// Parameters
 		final var body = Category.create().withName("name");
+
+		//Mock
+		when(metadataServiceMock.createCategory("my.namespace", "2281", body)).thenReturn(body.getName());
 		
 		// Call
 		webTestClient.post().uri("my.namespace/2281/metadata/categories")
 			.contentType(APPLICATION_JSON)
 			.bodyValue(body)
 			.exchange()
-			.expectStatus().isEqualTo(HttpStatus.NOT_IMPLEMENTED);
+			.expectStatus().isEqualTo(HttpStatus.CREATED);
```

Comment:
Change to `.expectStatus().isCreated()`? Also verify contenttype and that location is present with correct format in header (see MetadataStatusResourceTest for example how its done).